xtas/tasks/es.py

```diff
@@ -100,32 +139,58 @@ def fetch_query_batch(idx, typ, query, field='body'):
     hits = scan(_es(), {'query': query}, index=idx, doc_type=typ)
     return [hit['_source'][field] for hit in hits if field in hit['_source']]
 
-CHECKED_MAPPINGS = set()
```

Comment:
Does this cache disappear between sessions, while the types remain in the ES instance? Does it really help that much, considering that it's only used after a (presumably somewhat lengthy) calculation? Should it be named _CHECKED_MAPPINGS?